apps/next/pages/pricing.tsx

```diff
@@ -149,7 +149,7 @@ const Page: NextPage<Props> = ({ products }) => {
                 <ModalContent>
                     <ModalHeader>
                         <Center>Almost there!</Center>
-                        <Center>Connect to Twitter to continue.</Center>
+                        <Center>Connect to all platforms to continue.</Center>
```

Comment:
I think you only have to connect to Twitter for the pricing modal

Reply:
We prompt for both

Reply:
Oh you're right for the pricing